File: DB/src/main/java/adapter/converter/userProto/ConverterUser.java
package adapter.converter.userProto;

import Domain.Model.User;
import GRPC.proto.File.*;

public class ConverterUser
{
  public static UserProto User_To_UserProto(User user)
  {
    return UserProto.newBuilder()
        .setId(user.getId())
        .build();
  }


  public static User UserProto_To_User(UserProto userProto)
  {
    return new User(userProto.getId());
  }
}

File: Logic/AdapterToGRPC/Converter/ConverterUser.cs
using GRPC.Proto;
using Shared.Model;

namespace Logic.AdapterToGRPC;

public class ConverterUser
{
    public static User UserProtoToUser(UserProto userProto)
    {
        User user = new User();
        user.Id = userProto.Id;
        
        return user;
    }

    public static UserProto UserToUserProto(User user)
    {
        return new UserProto {Id = user.Id};
    }
}